hermes/probe/mock/mock_gcs.go

```diff
@@ -0,0 +1,88 @@
+package mock
```

Comment:
Nit: It's not a mock. It's a fake.
This article explains the difference between fakes, mocks and stubs: https://blog.pragmatists.com/test-doubles-fakes-mocks-and-stubs-1a7491dfa3da.

I would call this package `fakegcs`.

Reply:
Thank you! Changed to fakegcs